### DISCUSSION TITLE
Problema shelly dopo riavvio router

### DISCUSSION THREAD

**Discussion by ajeje-b** [indomus/forum#660] [Networking]
Ciao a tutti,

ho un problema con i miei dispositivi shelly (1, 2.5, dimmer2, PM e EM) in quanto risultano non disponibili su HA dopo un riavvio di router, switch e access point. Sono collegati tramite integrazione nativa, non MQTT.
Questa la situazione su HA per tutti i dispositivi:
![image](https://user-images.githubusercontent.com/72555385/131318345-5a73fe6e-c955-4104-8eca-92385c304312.png)
Il log dice: Config entry 'shelly1pm-xxxxxx' for shelly integration not ready yet; Retrying in background

Tutti risultano regolarmente connessi alla rete e riesco a controllarli con l'app shelly. 
Qualcuno ha idea del motivo o di una possibile risoluzione? Vorrei evitare di eliminarli e riaggiungerli visto che sono quasi 30 e soprattutto non vorrei doverlo fare ad ogni riavvio della rete.

Grazie mille in anticipo!

**Comment by snakuzzo** [comment]
Rimangono a vita così oppure dopo un po' tornano online?
Se restano così, apri una issue su github sul progetto di Core

**Reply by ajeje-b** [reply]
Grazie per la risposta. Ho provato ad aspettare una notte e riavviare il raspberry ma non sono mai tornati online. Provo ad aprire un issue allora!

**Reply by 2Max2Pax** [reply]
Hai provato a fare il reboot dal l'interfaccia web?
Anche a me succede di tanto in tanto su uno Shelly 2.5 e con il reboot risolvo.
Un'altra cosa prova a togliere corrente e poi ridarla alla linea su cui sono installati, usa il tastino del differenziale, non abbassare brutalmente la levetta.

**Reply by ajeje-b** [reply]
Ho provato il riavvio ma senza successo, tra qualche giorno rientro a casa e provo a togliere corrente. Grazie!
